boards/nrf52840dongle/Makefile.include

```diff
@@ -0,0 +1,27 @@
+# This board uses the vendor's serial bootloader
+
+PROGRAMMER ?= nrfutil
+
+ifeq (nrfutil,$(PROGRAMMER))
+
+  # Using nrfutil implies using the Nordic bootloader described at
+  # <https://devzone.nordicsemi.com/nordic/short-range-guides/b/getting-started/posts/nrf52840-dongle-programming-tutorial>.
+  #
+  # It has a static MBR at the first 4k, and an ample 128k Open USB Bootloader at
+  # the end, leaving 892k for the application. This overwrites any SoftDevice,
+  # but that's what the minimal working example for the dongle does as well.
+
+  ROM_OFFSET = 0x1000
+  FW_ROM_LEN = 0xdf000
```

Comment:
Remove this line to fix the remaining Murdock issues.

Reply:
And replace it with

    SLOT0_LEN = (0xdf000/2)

?

Reply:
Aehm, as I understand that would not only lead to things not working any more (as the code now thinks it's starting at 0x0 but is flashed to 0x1000, and we're not all PIE yet, are we?), but also (by not limiting FW_ROM_LEN) risk building overly large images that either will fail to flash or worse will overwrite the high portion of the bootloader.

Reply:
I don't think so because it's handled automatically by the build system (can we stop guessing things and move to the point where this can be merged ?)

Reply:
See [here](https://github.com/RIOT-OS/RIOT/blob/master/cpu/cortexm_common/ldscripts/cortexm_rom_offset.ld#L24).

Reply:
I just tried, flashing succeeded but the board won't come up. Going for the the blacklisting solution, this should make it passable. (The game of whether this declares its memory wrong or whether the suit/riotboot can't work on exotic-memory-layout bootloaders can be had outside this issue.)

Reply:
I think you should leave the `ROM_OFFSET` in place.

Reply:
OK, only the FW_ROM_LEN, that *does* make sense if the information can be passed around somewhere else (eg. in ROM_LEN).

Thanks, that appears to fix it; riotboot and suit example now build, pushing it to murdock.